highlight_view.py

```diff
@@ -692,12 +696,27 @@ def open_tooltip(view, point, line_report=False):
     if not errors:
         return
 
+    def on_navigate(href: str) -> None:
+        if href == "copy":
+            sublime.set_clipboard(join_msgs_raw(errors))
+            sublime.active_window().status_message("SublimeLinter: info copied to clipboard")
```

Comment:
In stead of taking the active window here, you can get the window the view belongs to. I don’t have the API docs at hand right now, bet you can find that easily. I think that’s a bit cleaner since the pop up belongs to a view and you act on the view on the next line.

Reply:
Good suggestion, I just made that change!